config/settings.js

```diff
@@ -27,11 +35,19 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
```

Comment:
I assume you'll also verify the final form in a running deployment.

On Tue, Jan 26, 2016 at 6:03 PM, Alex Meng <notifications@github.com> wrote:

> In config/settings.js
> <https://github.com/spinnaker/spinnaker/pull/702#discussion_r50916142>:
>
> > @@ -27,11 +35,19 @@
> >   */
> >
> >  window.spinnakerSettings = {
> > +  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
>
> Actually, looking at deck, I think this is just a fallback list in case no
> cloud providers are enabled. It seems like this can probably just be
> removed.
>
> See code
> <https://github.com/spinnaker/deck/blob/fee4c9839a465a614c105a52267e3ccb9f0e63e5/app/scripts/modules/core/account/account.service.js#L71>
> and test
> <https://github.com/spinnaker/deck/blob/0a9ed6034363ee8c1c75958822b4e9009f6781b9/app/scripts/modules/core/account/account.service.spec.js#L130>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/spinnaker/pull/702/files#r50916142>.
>